scripts/ci/install.sh

```diff
@@ -49,30 +49,23 @@ if [ "$ARGV_OPERATING_SYSTEM" == "linux" ]; then
     -r "$ARGV_ARCHITECTURE" \
     -s "$(pwd)" \
     -c "make info && make electron-develop"
+elif [ "$ARGV_OPERATING_SYSTEM" == "win32" ]; then
+  ./scripts/build/check-dependency.sh choco
+  choco install jq
+  choco install curl
 else
-  PIP_COMMAND=pip
-  if [ "$ARGV_OPERATING_SYSTEM" == "darwin" ]; then
-    ./scripts/build/check-dependency.sh brew
-    brew install ccache jq --force-bottle
-    PIP_COMMAND=pip2
-  elif [ "$ARGV_OPERATING_SYSTEM" == "win32" ]; then
-    ./scripts/build/check-dependency.sh choco
-    choco install jq
-    choco install curl
-  else
-    echo "Unsupported operating system: $ARGV_OPERATING_SYSTEM" 1>&2
-    exit 1
-  fi
+  echo "Unsupported operating system: $ARGV_OPERATING_SYSTEM" 1>&2
+  exit 1
```

Comment:
Oh, right – as @lurch mentioned; this should still support "darwin", even if there's no action to be taken.

Reply:
Done!